catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -165,7 +165,7 @@
     </feature>
 
     <feature name="catalog-confluence-source" version="${project.version}"
-             description="Confluence source and transformer">
+             description="Confluence source, config mapping provider, and transformer">
```

Comment:
✏️ Maybe we shouldn't be changing those since the provider is just an internal adapter to the configuration and not a new functionality of the feature.

Reply:
👍